mcm/Dockerfile

```diff
@@ -0,0 +1,42 @@
+# Angular templates
+FROM node:20.11.0-alpine3.19@sha256:9b61ed13fef9ca689326f40c0c0b4da70e37a18712f200b4c66d3b44fd59d98e AS frontend
+WORKDIR /usr/app
+
+# Copy the static files
+COPY ./scripts ./scripts
+COPY Gruntfile.js .
+COPY package.json .
+
+RUN npm install
```

Comment:
I had to pass `--legacy-peer-deps` back in the days when I made mine, is this no longer needed?

Reply:
No, it was not required because of the `npm` version used (10.2.4). Some details [here](https://stackoverflow.com/questions/66239691/what-does-npm-install-legacy-peer-deps-do-exactly-when-is-it-recommended-wh)